orchagent/portsorch.cpp

```diff
@@ -1441,6 +1499,20 @@ bool PortsOrch::setPortAdminStatus(Port &port, bool state)
     return true;
 }
 
+void PortsOrch::setHostTxReady(sai_object_id_t portId, const std::string &status)
+{
+    Port p;
+
+    if (!getPort(portId, p))
+    {
+        SWSS_LOG_ERROR("Failed to get port object for port id 0x%" PRIx64, portId);
+        return;
+    }
+
+    SWSS_LOG_NOTICE("Setting host_tx_ready status = %s, port_id = 0x%" PRIx64, status.c_str(), portId);
```

Comment:
@noaOrMlnx please log the logical port name (port.m_alias) as well for easy identification

Reply:
Done

Reply:
@noaOrMlnx please add the logical port name as done previously `port.m_alias.c_Str()`

Reply:
@prgeor Done